common/culture/cultures/orc.txt

```diff
@@ -923,9 +859,6 @@
 	}
 
 	moknathal = {
```

Comment:
I think Mok'nathal should have their own model. They are described as taller and bigger than Orcs (https://wow.gamepedia.com/Mok%27nathal#Appearance)

If you look at Rexxar new model without the mask on, he has very distinct Ogre features that none of the Orcs have:
<details>

![image](https://user-images.githubusercontent.com/47816228/103791086-99b2ea80-5074-11eb-8f77-4c7d4fa5deec.png)
</details

Reply:
I plan them later once the ogres are done.